.github/workflows/ci.yml

```diff
@@ -39,5 +51,3 @@ jobs:
       run: scripts/xcode_select_current_version.sh
```

Comment:
if we already did xcdoeselect_current_version.sh above why are we running this agin?

Reply:
I have managed to get two successive builds successful. All jobs are executed on same machine. I have removed the duplicate call to xcode selection.